src/pages/Home/Home.test.tsx

```diff
@@ -17,6 +17,11 @@ test("renders a log out button within a header", () => {
   );
 });
 
+test("renders a log out button at the end of the header", () => {
+  const { container } = render(<Home />);
+  expect(container.querySelector("ion-buttons")).toMatchSnapshot();
```

Comment:
Kinda iffy about this. Snapshot test don't often convey whether a change that breaks the test is an acceptable one or not. That's why keeping them as small as possible is usually the best idea.

In the above snapshot I confined the snapshot to the ion-buttons element, when in fact the actual DOM looks like this:
```jsx
  <ion-header>
      <ion-toolbar>
        <ion-title>
          Home
        </ion-title>
        <ion-buttons
          slot="end"
        >
          <ion-button>
            Log out
          </ion-button>
        </ion-buttons>
      </ion-toolbar>
    </ion-header>
```